krabcake/rs_hello/src/lib.rs

```diff
@@ -145,7 +145,7 @@ static mut THREAD_ID: c_uint = 0;
 static mut CTX: Context = Context {
     normalize_output: false,
 };
-static mut COUNTER: Tag = Tag(0);
```

Comment:
Hmm.

We should either create a new wrapper type for the purely numeric tags, or we should make this COUNTER carry a u64 rather than a Tag. 

There's no reason for the COUNTER to be a Tag itself, because it can never be anything except for one of the monotonically increasing integers.